*** sound/soc/bcm/Kconfig.orig	2017-10-30 13:14:25.269133066 -0400
--- sound/soc/bcm/Kconfig	2017-10-30 13:14:55.780582513 -0400
***************
*** 189,194 ****
--- 189,201 ----
  	help
  	  Say Y or M if you want to add support for Allo DigiOne.
  
+ config SND_BCM2708_SOC_MONOME
+        tristate "Support for monome-snd"
+        depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
+        select SND_SOC_CS4270
+        help
+        Say Y or M if you want to add support for Monome soundcard
+ 
  config SND_PISOUND
  	tristate "Support for Blokas Labs pisound"
  	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
